website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -958,6 +960,14 @@ A `gmsa` block supports the following:
 
 ---
 
+A `node_provisioning_profile` block supports the following:
```

Comment:
Alphabetical ordering of fields:
```markdown
  * `default_node_pools` - (Optional) Specifies whether default node pools should be provisioned automatically. Possible values are `Auto` and `None`. Defaults to `Auto`. At least one of `mode` or `default_node_pools` must be specified.

  * `mode` - (Optional) Specifies the provisioning mode for node pools created in this cluster. Possible values are `Auto` and `Manual`. Defaults to `Manual`. At least one of `mode` or `default_node_pools` must be specified.
 ```
Reference: [reference-documentation-standards](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-documentation-standards.md)

Reply:
Updated.